src/date/index.ts

```diff
@@ -27,3 +31,344 @@ export function getMonthAbbrByIndex(monthNumber: number): string {
 
   return monthNames[ monthNumber - 1 ];
 }
+
+
+/**
+ * This function converts date input coming from backend (generally in long format (ex. '2022-02-01T12:16:13'))
+ * to a specified format provided in 2nd paramter
+ * It converts all types of dates accepted by dayjs
+ *
+ * @param {Date} longDateValue - Date input to be converted to another format. This generally comes from backend and is
+ * in long format. For example '2022-02-01T12:16:13'
+ * @param {string} dateFormat - Format in which the date needs to be converted to.
+ *
+ * @example
+ * ```
+ * getDateFromLongValue('2022-02-01T12:16:13', 'DD MMM, hh:mm A');  // Output will be '01 Feb, 12:16 PM'
+ * getDateFromLongValue('2022-01-28T12:54:40', 'DD MMM, hh:mm A');  // Output will be '28 Jan, 12:54 PM'
+ * getDateFromLongValue('2022-01-25T12:08:12', 'DD MMM YYYY,  hh:mm A);  // Output will be '25 Jan 2022,  12:08 PM'
+ * getDateFromLongValue('2021/11/20', 'YYYY-MM-DD');  // Output will be '2021-11-20'
+ * getDateFromLongValue('21 Nov 2020', 'YYYY-MM-DD');  // Output will be '2020-11-21'
+ * getDateFromLongValue('Fri Feb 04 2022 15:24:28 GMT+0530 (India Standard Time)', 'YYYY-MM-DD');  // Output will be '2022-02-04'
+ * ```
+ */
+export function getDateFromLongValue(longDateValue: Date, dateFormat: string = 'DD MMM YYYY') {
```

Comment:
Are we going forward with the same name for this function?

Reply:
name changed